### PR TITLE
add id to user create event

### PR DESCRIPTION
**Pull request by nk-coding**
https://frontend.gropius.duckdns.org/components/9de999cf-58cd-4b47-aa0e-860cd2b6869c/issues/c117e3af-00b5-46ce-81b1-ef4591148d81

- [ ] Requirements of the issue are met
- [x] Code has been reviewed
- [ ] Code is documented
- [ ] GraphQL related artefacts are documented

### CODE REVIEW

File: spi-extension/src/main/java/org/misarch/keycloak/event/provider/DaprEventListenerProvider.java [hunk 1]
```diff
@@ -99,11 +99,13 @@ private void handleUserRegister(final String realmId, final String userId) {
         KeycloakModelUtils.runJobInTransaction(session.getKeycloakSessionFactory(), session -> {
             final RealmModel realm = session.realms().getRealm(realmId);
             final UserModel user = session.users().getUserById(realm, userId);
-            log.infof("Registered user [name=\"%s\", email=\"%s\", first name=\"%s\", last name=\"%s\"]", user.getUsername(), user.getEmail(), user.getFirstName(), user.getLastName());
+            log.infof(
+                    "Registered user [name=\"%s\", email=\"%s\", first name=\"%s\", last name=\"%s\"]",
+                    user.getUsername(), user.getEmail(), user.getFirstName(), user.getLastName());
```
Comment (KiLanger) [inline spi-extension/src/main/java/org/misarch/keycloak/event/provider/DaprEventListenerProvider.java:104]:
```
 log.infof(
                    "Registered user [name=\"%s\", email=\"%s\", first name=\"%s\", last name=\"%s\"]",
                    user.getUsername(), user.getEmail(), user.getFirstName(), user.getLastName()
);
```

In the docs we used prettier, which would request this. Not a blocking comment ;)

**Review by KiLanger — COMMENTED** [review]
Only linting

File: spi-extension/src/main/java/org/misarch/keycloak/event/provider/DaprEventListenerProvider.java (hunk 1 above)
Comment (delvh) [reply spi-extension/src/main/java/org/misarch/keycloak/event/provider/DaprEventListenerProvider.java:104]: That's java, not prettier.
I'd prefer formatting Java as Java and not as ~~uglier~~prettier.

**Review by delvh — APPROVED** [review]

File: spi-extension/src/main/java/org/misarch/keycloak/event/provider/DaprEventListenerProvider.java (hunk 1 above)
Comment (nk-coding) [reply spi-extension/src/main/java/org/misarch/keycloak/event/provider/DaprEventListenerProvider.java:104]: I know prettier does it differently
I just used the first formatter I could live with, but I also dislike the formatting here
just not enough to change the formatter though

**Pull request merged**